tests/utils/test_helpers.py

```diff
@@ -52,6 +52,7 @@ def signal_handler(signum, frame):
     def _parent_of_ignores_sigterm(parent_pid, child_pid, setup_done):
         def signal_handler(signum, frame):
             pass
+
```

Comment:
Could you revert the changes to this file please?

Reply:
Changes incorporated